includes/blocks.inc

```diff
@@ -422,7 +422,7 @@ function islandora_solr_simple_search_form($form, &$form_state) {
   $form['simple']["islandora_simple_search_query"] = array(
     '#size' => '15',
     '#type' => 'textfield',
-    '#title' => '',
+    '#title' => 'Search For:',
```

Comment:
I'm not 100% sure matching between the blocks is appropriate, though I may be focusing too much on the technical side: In the Advanced Search, it does map more directly to individual terms(/phrases), whereas the simple search is intended to be more free-form... That said, I would be fine with "Search term".

Visibility: I don't feel strongly either way... May want to make it especially explicit in the ticket whichever way, as being deployed to existing sites could cause hiccups in themes and the like developed without the title populated.